docs/search/criteria_reference/isbookmarked_criterion.md

```diff
@@ -0,0 +1,45 @@
+---
+description: IsBookmarked Search Criterion
+---
+
+# IsBookmarked Criterion
+
+The [`IsBookmarked` Search Criterion](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-Values-Content-Query-Criterion-Location-IsBookmarked.html)
+searches for location based on whether it's bookmarked or not.
```

Comment:
Bookmarked by anyone or bookmarked by the current user?

Reply:
According to [its implementation](https://github.com/ibexa/core/blob/main/src/lib/Search/Legacy/Content/Location/Gateway/CriterionHandler/Location/IsBookmarked.php#L70), it's about the current user.

```suggestion
searches for location based on whether it's bookmarked or not by the current user.
```